### PR TITLE
Add one body integral access

### PR DESCRIPTION
**Pull request by JoshRackers**
## Description
Added the one line `ao_multipole_potential` function to `export_mints.cc`. This allows the user to access one-center integrals from the python interface. The most common use case for this is to calculate electrostatic potential and field from an auxiliary basis set expansion of the electron density.

## User API & Changelog headlines
- added `ao_multipole_potential` to `export_mints.cc`

## Dev notes & details

- Makes no changes to existing `export_mints.cc`
- Adds a function to grab One Body Integrals for a given basis set.

## Checklist
- [x] [All or relevant fraction of full tests run]

## Status
- [x] Ready for review
- [x] Ready for merge

### CODE REVIEW

**Review by JonathonMisiewicz — APPROVED** [review]

**Review by loriab — APPROVED** [review]

**Review by maxscheurer — COMMENTED** [review]
Just one minor comment ☺️

**Comment by JonathonMisiewicz** [comment]
@JoshRackers Can you fix up the issues that Max mentioned? It should be quick.

File: psi4/src/export_mints.cc [hunk 1]
```diff
@@ -1385,7 +1388,9 @@ void export_mints(py::module& m) {
         .def("ao_multipoles", &IntegralFactory::ao_multipoles,
              "Returns a OneBodyInt that computes arbitrary-order AO multipole integrals", "order"_a, "deriv"_a = 0)
         .def("so_multipoles", &IntegralFactory::so_multipoles,
-             "Returns a OneBodyInt that computes arbitrary-order SO multipole integrals", "order"_a, "deriv"_a = 0)
+          "Returns a OneBodyInt that computes arbitrary-order SO multipole integrals", "order"_a, "deriv"_a = 0)
+        .def("ao_multipole_potential", &IntegralFactory::ao_multipole_potential,
+             "Returns a OneBodyInt that computes the potential and its derivatives up to specified order (maximum of 3)", "order"_a, "deriv"_a = 0)
```
[the remark this answers is not in the document]
Comment (JoshRackers) [reply psi4/src/export_mints.cc]: I didn't that you had added the `ao_multipoles` function as well! I will keep this as-is like you suggest. Good to know the integrals are now arbitrary order. I've updated the docstring.

**Comment by JoshRackers** [comment]
@JonathonMisiewicz I just updated the docstring. Thanks for the poke!

**Review by maxscheurer — APPROVED** [review]

**Pull request merged**